test/e2e/daemon_restart.go

```diff
@@ -183,8 +183,7 @@ func getContainerRestarts(c *client.Client, ns string, labelSelector labels.Sele
 	return failedContainers, containerRestartNodes.List()
 }
 
-// Flaky issues #17829, #19023
-var _ = Describe("DaemonRestart [Disruptive] [Flaky]", func() {
+var _ = Describe("DaemonRestart [Disruptive]", func() {
```

Comment:
Ah, you mean don't restart things. I'd assumed disruptive would include all sorts of netsplit, storage loss etc but more likely that it more or less translates to reboot and daemon restart today.